README.md

```diff
@@ -41,6 +41,8 @@ s3d .s3.production
 s3d .s3.staging
 s3d .s3.local
 ```
+# pass in custom author name
```

Comment:
i would add a note in the read me that the author name is used to name the nested folder for the uploaded files